web/html/src/components/dialog/ModalLink.tsx

```diff
@@ -21,11 +21,9 @@ export function ModalLink(props: Props) {
   const margin = props.text ? "" : " no-margin";
   var icon = props.icon && <i className={"fa " + props.icon + margin} />;
 
-  // TODO: Using `href="#"` is bad for accessibility and semantics, this should be a `<button>` that looks like it needs to
   return (
-    <a
+    <button
       id={props.id}
-      href="#"
       title={props.title}
       className={props.className}
```

Comment:
Is this perhaps missing the `btn-link` className? Maybe it is inherited but, just to make sure, I'd rather add it here. Am I missing something?

Reply:
Nice catch, in some cases the `btn-link` className were not inherited.

@ncounter I've updated PR accordingly, please take another look.